neo.UnitTests/UT_NeoToken.cs

```diff
@@ -29,6 +29,15 @@ public void TestSetup()
             Store = TestBlockchain.GetStore();
         }
 
+        [TestMethod]
+        public void Check_AssetDescription()
```

Comment:
I think it is not related to native contracts.

Reply:
Other ut class?

Reply:
`UT_AssetDescription`?